test/index.test.js

```diff
@@ -334,5 +356,68 @@ describe('module-resolver', () => {
                 transformerOpts,
             );
     });
+
+    describe('unknown filename', () => {
```

Comment:
In this scenario the `filename` is missing from the transformer options, and so the starting point for `.babelrc` lookup will be the cwd. That's why I thought "unknown filename" was an appropriate name for the test, but it may be lacking. Any suggestions?